src/pages/Character.tsx

```diff
@@ -131,22 +128,34 @@ const Character = () => {
 
   const handleImageUpload = async (event) => {
     const file = event.target.files[0];
-    if (file) {
-      const storageRef = ref(storage, `images/${currentUser.uid}/${file.name}`);
-      try {
-        await uploadBytes(storageRef, file);
-        const fileURL = await getDownloadURL(storageRef);
-        setCharacterData((prevData) => ({ ...prevData, imageURL: fileURL }));
+    if (!file) {
+      console.error('No file selected.');
+      return;
+    }
 
-        if (currentUser) {
-          await setDoc(doc(db, 'characters', currentUser.uid), {
-            ...characterData,
-            imageURL: fileURL,
-          });
-        }
-      } catch (error) {
-        console.error('Error uploading file:', error);
+    if (!currentUser) {
+      console.error('User is not authenticated.');
+      alert('Musisz być zalogowany, aby przesłać plik.');
+      return;
+    }
```

Comment:
Jeżeli do wykonania tej akcji potrzebny jest stan zalogowania, to po co user wgl ma możliwość wypełniania tego formularza, LUB po co wgl ma dostęp do tej strony?